lib/svg-icon-registry.service.ts

```diff
@@ -19,28 +20,45 @@ export class SvgIconRegistryService {
 	constructor(private http:Http) {
 	}
 
-	loadSvg(url:string): Observable<SVGElement> {
-
+	loadSvg(url:string, symbolID:string = ''): Observable<SVGElement> {
+		const orgUrl = url;
+		url += `#${symbolID}`;
```

Comment:
This adds # to all Urls whether or not they are using sprite symbols.

I would prefer that the original Url is kept as url and the sprite or symbol Url is changed, so something like:
```typescript
const symbolUrl = url + `#${symbol}`;
```